### DISCUSSION TITLE
Don't have recovery key

### DISCUSSION THREAD

**Discussion by sleepechoes** [ente-io/ente#3004] [Q&A]
So I had to reset my phone and it's been a pain to login to things because everything is under ente Auth can I have a recovery key please

**Comment by greeeen-dev** [comment]
[Ente cannot recover your account if you don't have access to your password and recovery key](https://help.ente.io/photos/faq/security-and-privacy#what-happens-if-i-forget-my-password)

**Reply by sleepechoes** [reply]
I have already sent an email to the email I was alsoakng sure it was on GitHub too (email is nicknel2on@proton.me I sent it from)
